### PR TITLE
PULL_REQUEST_TEMPLATE: do not allow new formulae in homebrew-science

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Comment by iMichka** [comment]
CC @Homebrew/science @MikeMcQuaid @ilovezfs @fxcoudert

**Comment by sjackman** [comment]
I'm happy with pointing users to Homebrew/core for new formulae. We should not however dissuade the maintenance of existing formulae.

**Comment by iMichka** [comment]
It was ment as a gentle nudge to consider migrating that formula elsewhere.

**Comment by iMichka** [comment]
I'm fine with that. I'll remove the part in the update block.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,9 +5,10 @@
 - [ ] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
 - [ ] Built your formula locally prior to submission with `brew install <formula>`?
 
-### New formula: have you
+### New formula:
 
-- [ ] Written a sensible test? (The best test is to compile and run a sample program.)
+We will no more accept new formulae in homebrew-science, as this tap is being phased out.
```
Comment (jonchang) [inline .github/PULL_REQUEST_TEMPLATE.md]: s/no more/no longer/

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -5,9 +5,10 @@
...
+Please consider submitting your formulae to https://github.com/Homebrew/homebrew-core or host it in your own tap.
```
Comment (jonchang) [inline .github/PULL_REQUEST_TEMPLATE.md]: singular formula

add link to https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html

**Comment by jonchang** [comment]
Thanks @iMichka.

**Pull request merged**